### PR TITLE
#2 Create Entity Model

### PR DESCRIPTION
**Pull request by christianrd**
Created Domain project and Entity Model

### CODE REVIEW

File: TanitaTrakerNetCore/Domain/Entities/BaseEntity.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
+﻿using System;
+namespace Domain.Entities
+{
+    public abstract class BaseEntity
+    {
+        protected BaseEntity()
+        {
+            CreatedAt = DateTime.Now;
```
Comment (Willjobs94) [inline TanitaTrakerNetCore/Domain/Entities/BaseEntity.cs:8]: Change this to `DateTime.UtcNow`

File: TanitaTrakerNetCore/Domain/Entities/BaseEntity.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        /// Gets or sets a value indicating whether this <see cref="T:Domain.Entities.BaseEntity"/> is active.
+        /// </summary>
+        /// <value><c>true</c> if is active; otherwise, <c>false</c>.</value>
+        public bool IsActive { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether this <see cref="T:Domain.Entities.BaseEntity"/> is deleted.
+        /// </summary>
+        /// <value><c>true</c> if is deleted; otherwise, <c>false</c>.</value>
+        public bool IsDeleted { get; set; }
```
Comment (Willjobs94) [inline TanitaTrakerNetCore/Domain/Entities/BaseEntity.cs:33]: Remove this property, we will implement it later with a different approach

File: TanitaTrakerNetCore/Domain/Entities/BaseEntity.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    }
```
Comment (Willjobs94) [inline TanitaTrakerNetCore/Domain/Entities/BaseEntity.cs:34]: Add two more property to this model
```
        public int CreatedByUserId { get; set; }
        public User CreatedByUser { get; set; }
```

**Comment by christianrd** [comment]
Unsolved

**Pull request closed without merging**